reo/urls.py

```diff
@@ -44,5 +44,6 @@
     re_path(r'^loadprofile_chillerthermal_chiller_cop/?$', views.loadprofile_chillerthermal_chiller_cop),
     re_path(r'^absorption_chiller_defaults/?$', views.absorption_chiller_defaults),
     re_path(r'^schedule_stats/?$', views.schedule_stats),
+    re_path(r'^ground_conductivity/?$', views.ground_conductivity),
```

Comment:
The ground_conductivity endpoint seems like it would be better housed in the ghp app, but do we need to provide it as an endpoint at all? Seems like the REopt API should not be the (public) source for ground conductivity.

Reply:
I moved this into the /ghpghx app views and exposed via /ghpghx urls.